tests/perf-system/analyzer/analyzer.py

```diff
@@ -130,6 +130,57 @@ def customize_table(self, fields_list: List[str], values_list: List[List]):
             custom_table.add_row(val_row)
         return custom_table
 
+    def plot_commits(self, df_responses: pd.DataFrame, df_generator: pd.DataFrame):
+        """
+        For submitted requests of the type: N posts 1 commit, this function will
+        plot the number of posted versus committed messages
+        """
+        init_post_id = 0
+        init_time = 0
+        custom_header = "x-ms-ccf-transaction-id"
+        for head in df_responses.iloc[0]["rawResponse"].split("\n"):

```

Comment:
I would suggest using https://docs.python.org/3/library/http.client.html#http.client.parse_headers

Reply:
I used parse email_parser and json to parse better. thanks for the suggestion